index.html

```diff
@@ -405,8 +405,278 @@ <h4 class="title-small">Winning the talent war with a scalable tech skills platf
+                <div>
+                    <h5 class="footer-heading-text">Resources</h5>
+                    <div class="footer-sub-heading">
+                        <h6 class="footer-sub-heading-text">Download pluralsight</h6>
+                        <h6 class="footer-sub-heading-text">Events</h6>
+                        <h6 class="footer-sub-heading-text">Teach</h6>
+                        <h6 class="footer-sub-heading-text">Partners</h6>
+                        <h6 class="footer-sub-heading-text">Afiliate programs</h6>
+                        <h6 class="footer-sub-heading-text">pluralsight one.org</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+            <div class="footer-elements">
+                <div>
+                    <h5 class="footer-heading-text">Contact</h5>
+                    <div class="footer-sub-heading">
+                        <h6 class="footer-sub-heading-text">contact</h6>
+                        <h6 class="footer-sub-heading-text">Help center</h6>
+                        <h6 class="footer-sub-heading-text">IP whitelist</h6>
+                        <h6 class="footer-sub-heading-text">site-map</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+        </div>
+        <div class="footer-bottom-area">
+            <div class="footer-icons">
+                <div class="footer-icon-size">
+                    <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/desktop/Facebook_Icon.svg"
+                        alt="">
+                </div>
+                <div class="footer-icon-size">
+                    <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/desktop/Instagram_Icon.svg"
+                        alt="">
+                </div>
+                <div class="footer-icon-size">
+                    <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/desktop/Twitter_Icon.svg"
+                        alt="">
+                </div>
+                <div class="footer-icon-size">
+                    <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/desktop/LinkedIn_Icon.svg"
+                        alt="">
+                </div>
+                <div class="footer-icon-size">
+                    <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/desktop/YouTube_Icon.svg"
+                        alt="">
+                </div>
+            </div>
+            <div class="footer-buttons">
+                <div>
+                    <a class="footer-button-style" href="">Terms of use</a>
```

Comment:
You never called any other button with "-style" suffix before, why did you choose to call it this now? Naming should be consistence therefore, change it to "footer-button" or something like that.

Reply:
done